src/data.ts

```diff
@@ -147,10 +147,16 @@ export function isGraticuleGenerator(data: Partial<Data> | Partial<VgData>): dat
   return 'graticule' in data;
 }
 
-export type DataSourceType = 'raw' | 'main' | 'row' | 'column' | 'lookup';
-
-export const MAIN = 'main' as const;
-export const RAW = 'raw' as const;
+export enum DataSourceType {
+  Raw = 'raw',
+  Main = 'main',
+  Row = 'row',
+  Column = 'column',
+  Lookup = 'lookup',
+};
+
+export const MAIN = DataSourceType.Main as const;
```

Comment:
Now I'm wondering if we should stop exporting MAIN and RAW separately, since the whole enum now has a type name to refer to. So consumers of it should probably use `DataSourceType.Raw` instead of `RAW`. I'll make that change too.